packages/server/graphql/types/JiraServerIntegration.ts

```diff
@@ -136,7 +135,18 @@ const JiraServerIntegration = new GraphQLObjectType<{teamId: string; userId: str
           (projectKeyFilter) => IntegrationRepoId.split(projectKeyFilter).projectKey!
         )
 
-        const issueRes = await integrationManager.getIssues(queryString, isJQL, projectKeys)
+        // "first + 1" is a trick to make pagination works easier, see hasNextPage
```

Comment:
+1 This is more cryptic than helpful. I think "Request one extra item to see if there are more results" would be easier to understand in a couple of month.